Scripts/Game/ODIN/Editor/Entities/SCR_EditorManagerEntity.c

```diff
@@ -0,0 +1,86 @@
+- This entity handles **only barebone editor functionality**, like access rules, or opening and closing.
+- Everything else, like camera or menu handling, must be on specialized editor modes (SCR_EditorModeEntity) and components (SCR_BaseEditorComponent)!
+- Editor can have a limitied state, influenced by its modes and checked by IsLimited() function:
+ + Editor is **limited** if all its modes are.
+ + If at least one mode is unlimited, the editor is **unlimited** as well.
+ + The state influences behavior of certain components, e.g., photo mode camera movement may be restricted if the editor is limited.
+ + As a rule of thumb, unlimited state means that player can ***cheat*** with the editor.
+- Default editor manager prefab is defined in SCR_EditorManagerCore (config is Configs/Core folder).
+*/
+modded class SCR_EditorManagerEntity : SCR_EditorBaseEntity
+{
+	/*!
+	Functions to tell the server to move an entity
+	*/
+	void ODIN_TeleportEntityToPlayer(RplId sourceEntityId, RplId targetEntityId, int emptyTerrainAreaRadius)
+	{
+		Rpc(ODIN_Server_TeleportEntityToPlayer, sourceEntityId, targetEntityId, emptyTerrainAreaRadius);
+	}
+	
+	[RplRpc(RplChannel.Reliable, RplRcver.Server)]
+	protected void ODIN_Server_TeleportEntityToPlayer(RplId entityId, RplId targetEntityId, int emptyTerrainAreaRadius)
+	{		
+		// get entity from id 
+		RplComponent entityRpl = RplComponent.Cast(Replication.FindItem(entityId));
+		RplComponent targetRpl = RplComponent.Cast(Replication.FindItem(targetEntityId));
+		
+		if (!entityRpl || !targetRpl)
+			return;
+		
+		IEntity sourceEntity = entityRpl.GetEntity();
+		IEntity targetEntity = targetRpl.GetEntity();
+		
+		if (!sourceEntity || !targetEntity)
+			return;
+		
+		// If target is sitting in a vehicle, move player inside as well (Taken from SCR_PlayerSpawnPoint.c)
+		SCR_CompartmentAccessComponent compartmentAccessTarget = SCR_CompartmentAccessComponent.Cast(targetEntity.FindComponent(SCR_CompartmentAccessComponent));
+		IEntity vehicle = compartmentAccessTarget.GetVehicle();
+		if (vehicle)
+		{
+			SCR_CompartmentAccessComponent compartmentAccessPlayer = SCR_CompartmentAccessComponent.Cast(sourceEntity.FindComponent(SCR_CompartmentAccessComponent));
+			
+			// if we succeed, we return to avoid teleporting the unit after having moved him into the vehicle. If we fail, we just continue and TP player next to vehicle
+			if (compartmentAccessPlayer.MoveInVehicleAny(vehicle))
+				return;
+		}
+		
+		// Get world coordinates of player 
+		vector target_pos;
+		SCR_WorldTools.FindEmptyTerrainPosition(target_pos, targetEntity.GetOrigin(), emptyTerrainAreaRadius);
+		
+		// get transform for rotation etc. 
+		vector target_transform[4];
+		sourceEntity.GetTransform(target_transform);
+		
+		// use new position, but keep rotation 
+		target_transform[3] = target_pos;
+		
+		// call transform
+		ODIN_SetTransform(sourceEntity, target_transform);
+	}
+	
+	void ODIN_SetTransform(IEntity sourceEntity, vector transform[4])
```

Comment:
What's the point of this function? Why not just call `SCR_EditableEntityComponent.SetTransform` directly?